logger/options.go

```diff
@@ -23,35 +21,35 @@ func MaxBodySize(max int) Option {
 	}
 }
 
-// Prefix functional option defines log line prefix.
+// Prefix sets log line prefix.
 func Prefix(prefix string) Option {
 	return func(l *Middleware) {
 		l.prefix = prefix
 	}
 }
 
-// IPfn functional option defines ip masking function.
+// IPfn sets IP masking function. If ipFn is nil then IP address will be logged as is.
 func IPfn(ipFn func(ip string) string) Option {
 	return func(l *Middleware) {
 		l.ipFn = ipFn
 	}
 }
 
-// UserFn functional option defines user name function.
-func UserFn(userFn func(r *http.Request) (string, error)) Option {
+// WithUser triggers user name logging if userFn is not nil.
+func WithUser(userFn func(r *http.Request) (string, error)) Option {
```

Comment:
both WithUser and WithSubject are confusing because they not just turning on user and subj (inlike WithBody) but also passing fn. I'd suggest change it back to the original names